tests/integration/cattletest/core/test_docker.py

```diff
@@ -653,12 +664,12 @@ def _path_to_volume(volume):
     return mounted_path
 
 
-def _wait_until_stopped(container, admin_client, timeout=45):
+def _wait_until_stopped(container, internal_test_client, timeout=45):
```

Comment:
Well, nothing to be honest. I changed it to use the admin_client.  The _check_path function calls this function with the "internal_test_client"